src/GraphQL/GraphQLEventSource.cs

```diff
@@ -0,0 +1,49 @@
+using System.Diagnostics.Tracing;
+using GraphQL.Types;
+
+namespace GraphQL;
+
+/// <summary>
+/// EventSource events emitted from the project.
+/// </summary>
+[EventSource(Name = "GraphQL")]
+internal class GraphQLEventSource : EventSource
+{
+    public static readonly GraphQLEventSource Log = new();
+
+    [NonEvent]
```

Comment:
https://stackoverflow.com/questions/27058931/why-the-restriction-on-parameter-type-and-count-on-eventsource-methods

Reply:
message	ERROR: Exception in Command Processing for EventSource GraphQL: Unsupported type Schema in event source.

Reply:
Why don't we either (a) implement it like `SchemaCreated(Schema schema) => SchemaCreated(schema.GetType().Name);` or (b) add an extension method that does the same?  From the little I've read here, I don't think we should add a method marked `[NonEvent]` and then have it write an event.

Reply:
Just an example of doing some calculations. Actually both events were taken from my head without any analysis. I'm fine to add anything else to track GraphQL internals.

Reply:
I mean that I of course can pass `schema.GetType().Name` from the caller and remove two overloads.

Reply:
That is also acceptable.

Reply:
I removed overloads.